Fix bug where dbs are lost on restart (#1638)
github/vscode-codeql: extensions/ql-vscode/CHANGELOG.md

## [UNRELEASED]

> - Fix a bug where persisted databases may be lost if VS code is restarted while databases are being initialized. [#1638](https://github.com/github/vscode-codeql/pull/1638)

Reviewer: ```suggestion
- Fix a bug where databases may be lost if VS Code is restarted while the extension is being started up. [#1638](https://github.com/github/vscode-codeql/pull/1638)
```

I think this describes the impact in a way the user can understand a bit better as the average user may not know when databases are being initialized.

Author: Thanks. That sounds better.
